packages/eslint-config-schibsted-chai/index.js

```diff
@@ -4,5 +4,5 @@ module.exports = {
   env: {
     node: true
   },
-  extends: ["./chai.js"].map(require.resolve)
```

Comment:
why this change?

Reply:
then `eslint` will just see `./chai.js` and will have to figure out the path. simpler and clearer to give absolute path IMO

Reply:
The main benefit of using the require.resolve is that it forces resolution to the packages specified in this packages package.json instead of the package that eslint finds in the root node_modules. This isn't a concern for the relative path since it's relative to this package. (Although this wasn't one of the tests I did.)

I also made the call that having less code here was better. I don't know the speed differences of resolving the path here, versus it being resolved somewhere later, but I'm presume it's negligible in this use case.

Happy to revisit, but will merge as-is since I have your approval.